data/json/items/gun/shot.json

```diff
@@ -827,10 +827,10 @@
     "pocket_data": [ { "pocket_type": "MAGAZINE", "ammo_restriction": { "shot": 6 } } ]
   },
   {
-    "id": "slam_shotgun",
+    "id": "four_winds_shotgun",
```

Comment:
You'd need a migration entry to json/items/migration.json if you're going to change the ID.
```
{
    "id": "slam_shotgun",
    "type": "MIGRATION",
    "replace": "four_winds_shotgun"
  },
```

Reply:
Er, WHAT?
Wow, K.